src/libraries/Microsoft.Extensions.Caching.Memory/src/MemoryCache.cs

```diff
@@ -53,6 +53,13 @@ public MemoryCache(IOptions<MemoryCacheOptions> optionsAccessor!!, ILoggerFactor
                 _options.Clock = new SystemClock();
             }
 
+            if (_options.TrackStatistics)
+            {
+                _allStats = new();
```

Comment:
I'm updating our coding style guidelines in https://github.com/dotnet/runtime/pull/67061 to require the Type being used when the type isn't obvious from the left-hand side. (basically the converse of our `var` guidelines.)

```suggestion
                _allStats = new List<WeakReference<Stats>>();
```